@xen-orchestra/backups/Backup.js

```diff
@@ -38,7 +38,7 @@ const DEFAULT_VM_SETTINGS = {
   fullInterval: 0,
   healthCheckSr: undefined,
   healthCheckVmsWithTags: [],
-  maxMergedDeltasPerRun: 2,
+  maxMergedDeltasPerRun: Infinity,
```

Comment:
The setting in `xo-server/config.toml` can be removed with that changes.

Also, add `@xen-orchestra/backups` to `CHANGELOG.unreleased.md`.